signup.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>판다마켓 회원가입</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="utility.css">
+  <link rel="stylesheet" href="signup.css">
+</head>
+<body>
+  <header>
+    <div class="logo wrap">
+      <a href="/">
+        <img src="images/logo_lg.png" alt="판다마켓">
+      </a>
+    </div>
+  </header>
+  <main>
+    <div class="wrap">
+      <form class="signup-form">
+        <div class="input-group">
+          <label for="useremail" class="bold">이메일</label>
+          <input type="email" id="useremail" class="text-lg regular">
+        </div>
+        <div class="input-group">
+          <label for="nickname" class="bold">닉네임</label>
+          <input type="text" id="nickname" class="text-lg regular">
+        </div>
+        <div class="input-group">
+          <label for="password" class="bold">비밀번호</label>
+          <input type="password" id="password" class="text-lg regular">
+          <img src="images/btn_visibility_off.png" alt="눈감음">
+        </div>
+        <div class="input-group">
+          <label for="checkPassword" class="bold">비밀번호 확인</label>
+          <input type="text" id="checkPassword" class="text-lg regular">
+          <img src="images/btn_visibility_on.png" alt="눈뜸">
+        </div>
+        <button class="btn-large text-xl semibold">회원가입</button>
```

Comment:
💬 여담
button 태그의 type 속성을 명시하지 않으면 submit을 기본값으로 가지게 됩니다~
이러한 type 속성은 button 태그에 중요한 요소이므로 명시적으로 작성해주는 것을 추천드려요.

```suggestion
        <button class="btn-large text-xl semibold" type="submit">회원가입</button>
```

+type이 submit인 button의 경우 form에서 제출 이벤트가 발생하면, 해당 요소의 onclick 이벤트가 실행되게 됩니다.
이를 더 풀어서 말하자면 form 태그안의 input, button 과 같은 상호작용 요소에서 submit 이벤트, enter key 입력이 발생하면 해당 form 에서의 submit 이벤트가 실행된다는 것입니다.
자세한 것은 추후 학습 하시면서 배우실 것이니 우선은 button 태그의 type 속성을 명시적으로 작성하는 것이 좋다는 것만 기억해주시면 좋겠습니다~